cloudflare/src/framework/auth.rs

```diff
@@ -24,12 +22,3 @@ impl Credentials {
 pub trait AuthClient {
     fn auth(self, credentials: &Credentials) -> Self;
 }
-
-impl AuthClient for RequestBuilder {
```

Comment:
Self-comment: moved this into the blocking_api.rs that contains the implementation for a blocking client.